applications/butler/templates/deployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
               value: "/opt/lsst/butler/secrets/butler-gcs-creds.json"
             - name: S3_ENDPOINT_URL
               value: {{ .Values.config.s3EndpointUrl | quote }}
+            {{- range $name, $url := .Values.config.additionalS3EndpointUrls }}
+            - name: LSST_RESOURCES_S3_PROFILE_{{ $name }}
```

Comment:
Would it make sense to change `$name` to  `$name | upper ` or if this is parsed elsewhere feel free to ignore

Reply:
Yeah it's kind of a goofy setup but it's the case-sensitive name of the profile so it needs to be in lowercase.